editor/sidebar/last-revision/index.js

```diff
@@ -81,6 +81,10 @@ class LastRevision extends Component {
 		const { revisions } = this.state;
 		const lastRevision = revisions.length ? revisions[ 0 ] : null;
```

Comment:
Could we move this under the check and drop the ternary? 
There's also another ternary we could drop for the `href` prop.